zebrad/src/components/sync.rs

```diff
@@ -731,7 +732,8 @@ where
         // so the last peer to respond can't toggle our mempool
         self.recent_syncs.push_extend_tips_length(new_downloads);
 
-        self.request_blocks(download_set).await?;
+        let response = self.request_blocks(download_set).await;
+        Self::handle_response(response)?;
```

Comment:
This ignores temporary extendtips errors

Reply:
Thanks for the explanation comments.